### DISCUSSION TITLE
Claro Chile Operator, does not work, IMS no registered.  suggestions?

### DISCUSSION THREAD

**Discussion by Tintolios** [kyujin-cho/pixel-volte-patch#208] [Q&A]
I tried with a Simcard from Claro Chile, it cannot register IMS, I added IMS APN (upper and lower case letters).
VoWifi works on a Samsung S10+, but on the pixel it is not able to work.

It works with another operator of Chile  ----->  https://github.com/kyujin-cho/pixel-volte-patch/discussions/207

Simcard Claro Chile (Roaming Rogers Canada) 

Any suggestions?
@kyujin-cho 
Thanks 

![Screenshot_20231112-003113](https://github.com/kyujin-cho/pixel-volte-patch/assets/65043538/961730a8-4eb2-4b92-bc4a-1288ade00e6d)
![Screenshot_20231112-003122](https://github.com/kyujin-cho/pixel-volte-patch/assets/65043538/9a0105cd-8d62-42bc-bad6-9f224031b2e7)
![Screenshot_20231112-003132](https://github.com/kyujin-cho/pixel-volte-patch/assets/65043538/e2188537-7aa8-4033-bb2a-1ed416a80e58)
![Screenshot_20231112-003149](https://github.com/kyujin-cho/pixel-volte-patch/assets/65043538/ee1fdbc4-b269-49de-8432-cafeb638b736)
![Screenshot_20231112-003157](https://github.com/kyujin-cho/pixel-volte-patch/assets/65043538/42d4aa77-c948-4d1e-8d02-0af4e2b81fbf)
![Screenshot_20231112-003543](https://github.com/kyujin-cho/pixel-volte-patch/assets/65043538/53c3d01b-c07d-449d-9d55-4dc9d7091621)

**Comment by PontoZero23** [comment]
Hello! I'm a Claro BR user and I also have this problem with it not activating on my Pixel, I ended up discovering that here in Brazil it doesn't activate outside of the main states and Samsung devices, as you said your device is an S10+, so that's why it works your VoLTE
